fuse/data/datasets/sample_caching_audit.py

```diff
@@ -102,4 +102,10 @@ def audit(self, cached_sample, fresh_sample):
                 f"sample id in which this staleness was caught: {get_sample_id(fresh_sample)}\n"
                 'NOTE: if small changes between the saved cached and the live-loaded/processed sample are ok for your use case, you can set a tolerance epsilon like this: audit_diff_kwargs={"math_epsilon":1e-9}'
                 f"diff = {diff}"
+                f"Error! During AUDIT found a mismatch between cached_sample and loaded sample.\n"
```

Comment:
Can you please define the message above and then do something like (to avoid duplication):
msg + diff + msg